tests/transactional/disable_grub.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Disable Grub timeout by transactional update
+#
+# Maintainer: Ciprian Cret <ccret@suse.com>
```

Comment:
hm is this guy knows that he is maintainer ? :)

Reply:
oops :)